spec/lucky/action_rendering_spec.cr

```diff
@@ -52,9 +52,11 @@ end
 describe Lucky::Action do
   describe "rendering HTML pages" do
     it "render assigns" do
-      body = Rendering::Index.new(build_context, params).call.body
+      response = Rendering::Index.new(build_context, params).call
+      body = response.body
 
       body.should contain "Anything"
+      response.debug_message.to_s.should contain "Rendering::Index"
```

Comment:
I think this should test `"Rendering::IndexPage"` right? I may be misreading this though

Reply:
Definitely right.